subt/ros/robot/launch/k2.launch

```diff
@@ -1,96 +1,92 @@
+    </node>
 
-	<!-- REAL SENSE /-->	
-        <include file="$(find realsense2_camera)/launch/rs_rgbd.launch" if="$(arg record_sensors)">
-		<arg name="camera" 		value="camera"/>
-		<arg name="serial_no" value="920312070828"/>
-		<arg name="enable_fisheye"      value="false"/>
+    <!-- Remove noise -->
+    <node name="depth_denoiser_front" pkg="proxy" type="depth_filter">
+      <param name="max_support_distance" value="0.6"/>
+      <param name="min_support_fraction" value="0.75"/>
+      <param name="max_depth" value="10.0"/>
+      <remap from="depth_denoiser_front/input" to="front_depth/depth"/>
+      <remap from="depth_denoiser_front/output" to="front_depth/clean_depth"/>
+    </node>
+    <node name="depth_denoiser_rear" pkg="proxy" type="depth_filter">
+      <param name="max_support_distance" value="0.6"/>
+      <param name="min_support_fraction" value="0.75"/>
+      <param name="max_depth" value="10.0"/>
+      <remap from="depth_denoiser_rear/input" to="rear_depth/depth"/>
+      <remap from="depth_denoiser_rear/output" to="rear_depth/clean_depth"/>
+    </node>
 
-  		<arg name="json_file_path"      value="$(find robot)/params_common/d435i.json"/>
-		<arg name="depth_width"         value="848"/>
-  		<arg name="depth_height"        value="480"/>
-  		<arg name="enable_depth"        value="true"/>
+    <!-- Merge depth, camera_info and lidar streams. -->
+    <node pkg="topic_tools" type="relay" name="depth_merge_front" args="front_depth/clean_depth all_depth/depth"/>
+    <node pkg="topic_tools" type="relay" name="depth_merge_rear" args="rear_depth/clean_depth all_depth/depth"/>
+    <node pkg="topic_tools" type="relay" name="camera_info_merge_front" args="front_depth/camera_info all_depth/camera_info"/>
+    <node pkg="topic_tools" type="relay" name="camera_info_merge_rear" args="rear_depth/camera_info all_depth/camera_info"/>
+    <node pkg="topic_tools" type="relay" name="scan_merge_front" args="front_scan all_scan"/>
+    <node pkg="topic_tools" type="relay" name="scan_merge_rear" args="rear_scan all_scan"/>
+  </group>
 
-  		<arg name="enable_infra1"       value="false"/>
-  		<arg name="enable_infra2"       value="false"/>
+  <node pkg="proxy" type="traversability" name="local_map" output="screen">
+    <param name="publish_rate" value="0.2"/>
+    <param name="num_threads" value="3"/>
+    <param name="blind_columns_left" value="20"/>
+    <param name="depth_image_stride" value="1"/>
+    <param name="horizontal_pixel_offset" value="3"/>
+    <param name="vertical_pixel_offset" value="3"/>
+    <param name="synthetic_obstacle_distance" value="0.64"/>
+    <param name="visible_ground_min" value="0.3"/>
+    <param name="visible_ground_max" value="3.4"/>
+    <param name="max_bump_height" value="0.1"/>
+    <param name="max_slope" value="25"/>
+    <param name="depth_subsampling" value="143"/>
+    <!-- Six seconds worth of history. -->
+    <param name="max_depth_observations" value="60"/> <!-- 6 * 10 Hz -->
```

Comment:
Depth cameras are configured at 6 Hz, not 10 Hz.

Didn's we extend this to 12 seconds on Skiddy?